net/tshttpproxy/tshttpproxy_windows.go

```diff
@@ -221,30 +201,46 @@ type autoProxyOptions struct {
 // WINHTTP_PROXY_INFO
 // https://docs.microsoft.com/en-us/windows/win32/api/winhttp/ns-winhttp-winhttp_proxy_info
 type winHTTPProxyInfo struct {
-	AccessType  uint16
+	AccessType  uint32
 	Proxy       *uint16
 	ProxyBypass *uint16
 }
 
-var proxyForURLOpts = &autoProxyOptions{
+type winHGlobal windows.Handle
```

Comment:
why do we need this type?

Reply:
In terms of "will it compile," we don't, but `HGLOBAL` is considered to be distinct from `HANDLE` as far as the SDK is concerned. I wanted to reflect that.